test/run_tests.sh

```diff
@@ -26,10 +33,24 @@ if ! docker inspect "$builder_image"; then
 fi
 
 if docker build -f test/Dockerfile -t kudo-test .; then
-    if docker run -e INTEGRATION_OUTPUT_JUNIT --net=host -it -m 4g -v /var/run/docker.sock:/var/run/docker.sock -v "$(pwd)"/reports:/go/src/github.com/kudobuilder/kudo/reports --rm kudo-test make "$TARGET"; then
+    docker run -e INTEGRATION_OUTPUT_JUNIT --net=host -it -m 4g \
+        --name "$CONTAINER_NAME" \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v "$(pwd)"/reports:/go/src/github.com/kudobuilder/kudo/reports \
+        kudo-test make "$TARGET"
+    RESULT=$?
```

Comment:
I think this will never get run on failure, due to `errexit`.

Reply:
Oh no, looks I have to use traps then :(